superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -1188,18 +1218,7 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
                   dbName={dbName}
                   dbModel={dbModel}
                 />
-                {connectionAlert && (
-                  <StyledAlertMargin>
-                    <Alert
-                      closable={false}
-                      css={(theme: SupersetTheme) => antDAlertStyles(theme)}
-                      type="info"
-                      showIcon
-                      message={t('IP Allowlist')}
-                      description={connectionAlert.ALLOWED_IPS}
-                    />
-                  </StyledAlertMargin>
-                )}
+                {connectionAlert && renderStepTwoAlert()}
```

Comment:
I wanted to refactor this to be driven by `superset_text.yml`.

1. We'd update the superset_text schema to look like this:
```
DB_CONNECTION_ALERT:
    catalog:  # gsheets specific
         message: <add message here>
         description: <add description hear>
   host:  # basic mixin specific
        message: <add message here>
        description: <add description hear>
```

2. Based upon what is parameters come back in the `/available` endpoint and what the config has we'd render the alert accordingly.